book_haven_api/tests/requests/libraries/books/test_add_book_to_library.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+from django.urls import reverse
+from rest_framework.test import APIClient
+from book_haven_api.models import Library, Book
+
+@pytest.fixture
+def Libraries():
+    library1 =  Library.objects.create(
+        id=1,
+        name='Library1',
+        street='1234 Library1 St.',
+        city='Library1 City',
+        state='CA',
+        zip='12345',
+        lat=123.456,
+        lon=123.456,
+    )
+
+    return library1
+
+
+@pytest.mark.django_db
+def test_add_book_to_library(Libraries):
+    print('add_book_to_library_happy_path')
```

Comment:
In general, any linting system you use is going to flag this. You should be using the Python [`logging` package](https://docs.python.org/3/library/logging.html) to log things like this to some level - typically `INFO` if you want it to always be visible, or `DEBUG` if you only want it locally/in debug mode.

Reply:
This is really great info, will try to look into this.